### PR TITLE
[zh-cn] init translation for glossary value/variable

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Add glossary translation for:
* value
* variable

### CODE REVIEW

File: files/zh-cn/glossary/value/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 值
+slug: Glossary/Value
+---
+
+在数据或对象{{Glossary("Wrapper", "包装器")}}的概念下，值是对象包装器包含的{{Glossary("Primitive", "原始值")}}。在{{Glossary("Variable", "变量")}}或{{Glossary("Property", "属性")}}的概念下，值可以是原始类型或{{Glossary("Object reference", "对象引用")}}。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/value/index.md:6]: markdown bold syntax seems not to work here

File: files/zh-cn/glossary/value/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+在 CSS 属性值的概念下，存在指定值、计算值和实际值，应用于每个元素和伪元素的每个 CSS 属性的最终值是一个四步计算的结果：值由定义决定（[指定值](/zh-CN/docs/Web/CSS/specified_value)），然后解析成一个用于继承的值（[计算值](/zh-CN/docs/Web/CSS/computed_value)），然后在必要时转换为绝对的值（[应用值](/zh-CN/docs/Web/CSS/used_value)），最终根据本地环境的限制进行转化（[实际值](/zh-CN/docs/Web/CSS/actual_value)）。
+
+关于 CSS 的数据类型，详见 [CSS 值和单位](/zh-CN/docs/Web/CSS/CSS_Values_and_Units)。
+
+## 参见
+
+- 维基百科上的[原始值包装类](https://en.wikipedia.org/wiki/Primitive_wrapper_class)词条（英文）
```
Comment (jasonren0403) [inline files/zh-cn/glossary/value/index.md:14]: Haven't found the simplified Chinese wiki here

File: files/zh-cn/glossary/value/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/glossary/value/index.md:6]: Please still wrapper this with `bold`. The problem is caused by smartlink helper function in mdn/yari (when the link need to fallback to en-US).

File: files/zh-cn/glossary/value/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/glossary/value/index.md:14]: Yes, and let's remove the note `（英文）`

File: files/zh-cn/glossary/value/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/glossary/value/index.md:6]: Okay.

File: files/zh-cn/glossary/variable/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: 变量
+slug: Glossary/Variable
+---
+
+变量是{{Glossary("Value", "值")}}的命名参照。这样，不可预测的值可以通过预定的名称来访问。
```
Comment (yin1999) [inline files/zh-cn/glossary/variable/index.md:6]:
```suggestion
变量是{{Glossary("Value", "值")}}的命名引用。这样，不可预测的值可以通过预定的名称来访问。
```

File: files/zh-cn/glossary/value/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/glossary/value/index.md:6]:
```suggestion
在数据或对象**{{Glossary("Wrapper", "包装器")}}**的概念下，值是对象包装器包含的**{{Glossary("Primitive", "原始值")}}**。在**{{Glossary("Variable", "变量")}}**或**{{Glossary("Property", "属性")}}**的概念下，值可以是原始类型或**{{Glossary("Object reference", "对象引用")}}**。
```

File: files/zh-cn/glossary/value/index.md (hunk 2 above)
Comment (jasonren0403) [reply files/zh-cn/glossary/value/index.md:14]:
```suggestion
- 维基百科上的[原始值包装类](https://en.wikipedia.org/wiki/Primitive_wrapper_class)词条
```

File: files/zh-cn/glossary/value/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: 值
 slug: Glossary/Value
 ---
 
-在数据或对象{{Glossary("Wrapper", "包装器")}}的概念下，值是对象包装器包含的{{Glossary("Primitive", "原始值")}}。在{{Glossary("Variable", "变量")}}或{{Glossary("Property", "属性")}}的概念下，值可以是原始类型或{{Glossary("Object reference", "对象引用")}}。
+在数据或对象**{{Glossary("Wrapper", "包装器")}}**的概念下，值是对象包装器包含的**{{Glossary("Primitive", "原始值")}}**。在**{{Glossary("Variable", "变量")}}**或**{{Glossary("Property", "属性")}}**的概念下，值可以是原始类型或**{{Glossary("Object reference", "对象引用")}}**。
```
Comment (yin1999) [inline files/zh-cn/glossary/value/index.md:6]:
```suggestion
在数据或对象{{Glossary("Wrapper", "包装器")}}的概念下，值是对象包装器包含的{{Glossary("Primitive", "原始值")}}。在{{Glossary("Variable", "变量")}}或{{Glossary("Property", "属性")}}的概念下，值可以是原始类型或{{Glossary("Object reference", "对象引用")}}。
```

**Review by yin1999 — COMMENTED** [review]
It seems to be an old issue remarkjs/remark#908

Let's revert the changes temporary

**Review by yin1999 — APPROVED** [review]

**Pull request merged**